sql/sem/builtins: TestGetSSTableMetricsSingleNode failed

sql/sem/builtins.TestGetSSTableMetricsSingleNode [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12837516?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12837516?buildTab=artifacts#/) on master @ [caff15394fcbe37208b46b2973714c27cc3a1417](https://github.com/cockroachdb/cockroach/commits/caff15394fcbe37208b46b2973714c27cc3a1417):


```
=== RUN   TestGetSSTableMetricsSingleNode
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/ad72371e92981b2b572c54a0422f4d25/logTestGetSSTableMetricsSingleNode3464967091
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:146: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
    conditional_wrap.go:190: 
        pkg/sql/sem/builtins/generator_builtins_test.go:163: (TestGetSSTableMetricsSingleNode)
        	NOTICE: .GetFirstStoreID() called via implicit interface StorageLayerInterface;
        HINT: consider using .StorageLayer().GetFirstStoreID() instead.
    generator_builtins_test.go:200: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/sql/sem/builtins/generator_builtins_test.go:200
        	Error:      	Not equal: 
        	            	expected: 1
        	            	actual  : 2
        	Test:       	TestGetSSTableMetricsSingleNode
    panic.go:541: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/ad72371e92981b2b572c54a0422f4d25/logTestGetSSTableMetricsSingleNode3464967091
--- FAIL: TestGetSSTableMetricsSingleNode (32.23s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGetSSTableMetricsSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/sem/builtins.TestGetSSTableMetricsSingleNode [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12847707?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12847707?buildTab=artifacts#/) on master @ [58055a5ec733dff280ec24f9f65a390abdbbc695](https://github.com/cockroachdb/cockroach/commits/58055a5ec733dff280ec24f9f65a390abdbbc695):


```
=== RUN   TestGetSSTableMetricsSingleNode
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/ad72371e92981b2b572c54a0422f4d25/logTestGetSSTableMetricsSingleNode2620764746
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:146: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
    conditional_wrap.go:190: 
        pkg/sql/sem/builtins/generator_builtins_test.go:163: (TestGetSSTableMetricsSingleNode)
        	NOTICE: .GetFirstStoreID() called via implicit interface StorageLayerInterface;
        HINT: consider using .StorageLayer().GetFirstStoreID() instead.
    generator_builtins_test.go:200: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/sql/sem/builtins/generator_builtins_test.go:200
        	Error:      	Not equal: 
        	            	expected: 1
        	            	actual  : 2
        	Test:       	TestGetSSTableMetricsSingleNode
    panic.go:541: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/ad72371e92981b2b572c54a0422f4d25/logTestGetSSTableMetricsSingleNode2620764746
--- FAIL: TestGetSSTableMetricsSingleNode (39.26s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGetSSTableMetricsSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/sem/builtins.TestGetSSTableMetricsSingleNode [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12857887?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12857887?buildTab=artifacts#/) on master @ [22ace076cccf656dc8e73ca198068ff4176da0ce](https://github.com/cockroachdb/cockroach/commits/22ace076cccf656dc8e73ca198068ff4176da0ce):


```
=== RUN   TestGetSSTableMetricsSingleNode
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/ad72371e92981b2b572c54a0422f4d25/logTestGetSSTableMetricsSingleNode926387168
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:146: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
    conditional_wrap.go:190: 
        pkg/sql/sem/builtins/generator_builtins_test.go:163: (TestGetSSTableMetricsSingleNode)
        	NOTICE: .GetFirstStoreID() called via implicit interface StorageLayerInterface;
        HINT: consider using .StorageLayer().GetFirstStoreID() instead.
    generator_builtins_test.go:200: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/sql/sem/builtins/generator_builtins_test.go:200
        	Error:      	Not equal: 
        	            	expected: 1
        	            	actual  : 2
        	Test:       	TestGetSSTableMetricsSingleNode
    panic.go:541: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/ad72371e92981b2b572c54a0422f4d25/logTestGetSSTableMetricsSingleNode926387168
--- FAIL: TestGetSSTableMetricsSingleNode (40.84s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGetSSTableMetricsSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/sem/builtins.TestGetSSTableMetricsSingleNode [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12873609?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12873609?buildTab=artifacts#/) on master @ [b8c7824cda529e4aca05cb675a5cf500cdda7c0f](https://github.com/cockroachdb/cockroach/commits/b8c7824cda529e4aca05cb675a5cf500cdda7c0f):


```
=== RUN   TestGetSSTableMetricsSingleNode
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/ad72371e92981b2b572c54a0422f4d25/logTestGetSSTableMetricsSingleNode4024375518
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:146: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
    conditional_wrap.go:190: 
        pkg/sql/sem/builtins/generator_builtins_test.go:163: (TestGetSSTableMetricsSingleNode)
        	NOTICE: .GetFirstStoreID() called via implicit interface StorageLayerInterface;
        HINT: consider using .StorageLayer().GetFirstStoreID() instead.
    generator_builtins_test.go:200: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/sql/sem/builtins/generator_builtins_test.go:200
        	Error:      	Not equal: 
        	            	expected: 1
        	            	actual  : 2
        	Test:       	TestGetSSTableMetricsSingleNode
    panic.go:541: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/ad72371e92981b2b572c54a0422f4d25/logTestGetSSTableMetricsSingleNode4024375518
--- FAIL: TestGetSSTableMetricsSingleNode (38.14s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGetSSTableMetricsSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/sql/sem/builtins/generator_builtins_test.go
+++ b/pkg/sql/sem/builtins/generator_builtins_test.go
@@ -197,5 +197,5 @@ func TestGetSSTableMetricsSingleNode(t *testing.T) {
 		require.NotEqual(t, approximateSpanBytes, 0)
 		count++
 	}
-	require.Equal(t, 1, count)
+	require.GreaterOrEqual(t, count, 1)
 }

